stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -63,8 +63,8 @@ internal class PaymentSheetViewModel internal constructor(
     private val _startConfirm = MutableLiveData<ConfirmPaymentIntentParams>()
     internal val startConfirm: LiveData<ConfirmPaymentIntentParams> = _startConfirm
 
-    private val _viewState = MutableLiveData<ViewState>(null)
-    internal val viewState: LiveData<ViewState> = _viewState.distinctUntilChanged()
+    private val _viewState = MutableLiveData<ViewState.PaymentSheet>(null)
```

Comment:
Why is this `MutableLiveData` instantiated with value `null`, but in `PaymentOptionsViewModel` we have

```
private val _viewState = MutableLiveData<ViewState.PaymentOptions>(ViewState.PaymentOptions.Ready)
```

Reply:
This is because in PaymentSHeetViewModel we have to wait for the PaymentFlowResult before we enable the buy button.   PaymentOptions doesn't have this requriement.